src/scene/planar_scene_node.rs

```diff
@@ -285,6 +286,19 @@ impl PlanarSceneNodeData {
         self.set_texture(texture)
     }
 
+    /// Sets the texture of the objects contained by this node and its children.
+    ///
+    /// The texture is loaded from a byte slice and registered by the global `TextureManager`.
+    ///
+    /// # Arguments
+    ///   * `path` - relative path of the texture on the disk
+    #[inline]
+    pub fn set_texture_from_memory(&mut self, image_data: &[u8], name: &str) {
+        let texture = TextureManager::get_global_manager(|tm| tm.add_image(image::load_from_memory(image_data).expect("Invalid data"), name));
```

Comment:
The `tm.add_image(image::load_from_memory(image_data).expect("Invalid data"), name)` should be done by the texture manager too. For example by also adding a method `.add_image_from_memory()` to the texture manager.

Reply:
I decided to keep `TextureManager.add_image` public, so users have the ability to create `DynamicImage` themselves